examples/si1145_test_gains.py

```diff
@@ -0,0 +1,82 @@
+# SPDX-FileCopyrightText: 2017 Scott Shawcroft, written for Adafruit Industries
+# SPDX-FileCopyrightText: Copyright (c) 2022 Carter Nelson for Adafruit Industries
```

Comment:
I think we only need 1 copyright notice here instead of two.

Since it's a new file that you created now you can also update the year to 2024 and use your name or username instead of these ones. 

You can also omit the "for Adafruit Industries" which is intended to indicate development work that was paid for by Adafruit.